packages/eslint-plugin/package.json

```
@@ -56,8 +56,8 @@
     "generate:breaking-changes": "tsx tools/generate-breaking-changes.mts",
     "generate:configs": "npx nx generate-configs repo",
     "lint": "npx nx lint",
-    "test": "cross-env NODE_OPTIONS=\"--experimental-vm-modules\" jest --logHeapUsage",
-    "test-single": "cross-env NODE_OPTIONS=\"--experimental-vm-modules\" jest --no-coverage",
+    "test": "vitest run",
```

Let's align these with the other vitest PRs?
```suggestion
    "test": "vitest --run --config=$INIT_CWD/vitest.config.mts",
```

---

Do we know why? 

We should be able to run `vitest run` in all of them and auto discover the config

Is vitest not discovering mts config?